GUIDELINES.md

```diff
@@ -112,7 +112,13 @@ plugins for `eslint` in our builds.
 First, add an `.eslintrc.json` file to your project and put this in there.
 
     {
```

Comment:
Like I mentioned in the previous diff [here](https://github.com/bucharest-gold/entente/pull/113#discussion_r87890375), this could maybe become `.eslint.yaml`, in which case it could contain comments(!) and could look something like this:

```yaml
extends: semistandard
rules:
  prefer-const: error
  block-scoped-var: error
  no-use-before-define:
    - error
    - nofunc
```

Or with copious amounts of comments:
```yaml
extends: semistandard
rules:
  # Error if a let/var is never mutated -- must be changed to const
  prefer-const: error

  # Make var act like let
  block-scoped-var: error

  # Prevent ReferenceError due to "temporal dead zone"
  # For more info, see: http://eslint.org/docs/rules/no-use-before-define
  no-use-before-define:
    - error
    - nofunc # Ignore functions, as they're always hoisted
```